models/public/hrnetv2/model.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  HRNetV2-W48 is semantic-segmentation model based on architecture described in paper
+  High-Resolution Representations for Labeling Pixels and Regions <https://arxiv.org/abs/1904.04514>.
+  This is PyTorch implementation based on retaining high resolution representations throughout the model
+  and pretrained on ADE20k dataset. The model input is a blob that consists of a
+  single image of [1, 3, 320, 320] in RGB order.
+  For details about implementation of model, check out the Semantic Segmentation on MIT ADE20K dataset in PyTorch
+  <https://github.com/CSAILVision/semantic-segmentation-pytorch>.
+task_type: semantic_segmentation
+files:
+  - name: model/official_hrnet.py
+    size: 16557
+    sha256: b5a3df511c3ba6441f6a2d61df65feed50222875353ebc50535d3eb67877e35c
+    source: https://raw.githubusercontent.com/alexmasterdi/semantic-segmentation-pytorch/master/mit_semseg/models/official_hrnet.py
```

Comment:
not sure that it is good idea to download modified code from your repo.
@eizamaliev what do you think about it?
Possibly better to upload converted onnx model on download.01.org and here provide instruction how code should be modified for it.

Reply:
At this moment model downloaded from original repo. Also model success converted to IR format, so we don't need to upload converted onnx model on download.01.org.